managed/resource_cluster.go

```diff
@@ -493,6 +452,49 @@ type resourceCluster struct {
 }
 
 func EditBackupSchedule(ctx context.Context, backupScheduleStruct BackupScheduleInfo, scheduleId string, backupDes string, accountId string, projectId string, clusterId string, apiClient *openapiclient.APIClient) error {
+	if fflags.IsFeatureFlagEnabled(fflags.INCREMENTAL_BACKUP) {
```

Comment:
I think as we discussed instead of making calls and failing it makes sense to continue with the feature flag for now and remove it once the feature is enabled in prod later.